integration/test/test_multi_app.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+#
+#  Copyright (c) 2015 - 2023, Intel Corporation
+#  SPDX-License-Identifier: BSD-3-Clause
+#
+
+"""Test the use case of running multiple apps running simultaneously.
+The test will launch an instance of geopmbench and a non-MPI app and
+verify they're tracked/profiled correctly.
+
+"""
+
+import sys
+import unittest
+import subprocess
+import os
+
+import geopmpy.io
+from integration.test import util
+
+
+class TestIntegration_multi_app(unittest.TestCase):
+    TEST_NAME = 'test_multi_app'
+    TIME_LIMIT = 30
+    NUM_NODE = 1
+    EXPECTED_REGIONS = {'model-init', 'stream', 'dgemm'}
+
+    @classmethod
+    def setUpClass(cls):
+        sys.stdout.write('(' + os.path.basename(__file__).split('.')[0] +
+                         '.' + cls.__name__ + ') ...')
+        subprocess.run(['/bin/bash', 'test_multi_app.sh'],
```

Comment:
A simple solution to the PATH problem here is to assume that test_multi_app.sh script is in the same directory as the test_multi_app.py script.  The path to the script itself

os.path.join(os.path.dirname(os.path.abspath(__file__)), 'test_multi_app.sh') or something similar